### ISSUE TITLE
Progress overlay mismatch

### ISSUE THREAD

**Issue by Hrishikesh-K** [katspaugh/wavesurfer.js#2156]
**Wavesurfer.js version(s):** 4.4.0

**Browser and operating system version(s):** Chrome 87 (Windows 10)

**The problem:**

This is probably not a problem with Wavesurfer as it's working everywhere that I have checked, but I just can't get it working on my webpage. I'm stumped and would appreciate any help. However, my apologies if I shouldn't be asking the question here, feel free to close it.

I have added Wavesurfer to my webpage using the following code:

```JS
window.addEventListener('load', function() {
  var mediaA = document.querySelectorAll('.audio-player')
  if (mediaA != null) {
    mediaA.forEach(medium => {
      var wavesurfer = WaveSurfer.create({
        pixelRatio: 1,
        height: 75,
        barWidth: 3,
        normalize: true,
        container: medium,
        responsive: true,
        waveColor: "#eee",
        progressColor: "OrangeRed",
        cursorColor: "OrangeRed",
        partialRender: true
      })
      wavesurfer.load(medium.getAttribute('src'))
    })
  }
}
```

It seems to load fine, however, the progress colour is not overlaying correctly on top of the wave colour. It's happening something like this:

![image](https://user-images.githubusercontent.com/11403551/104798078-ad372180-57e9-11eb-90bf-226e582ccf1a.png)

**Use behaviour needed to reproduce the issue:**

This is happening here: https://6001c7436dcd4500085b1f78--hrishikeshk.netlify.app/projects/dropbox/#radio-ads. The Wavesurfer might take a few seconds to display, so kindly wait.

What am I missing? Kindly help.

**Comment by thijstriemstra** [comment]
> This is probably not a problem with Wavesurfer as it's working everywhere that I have checked, 

If you can't show a local example without extra user code that reproduces this problem, I'm afraid we can't help, because nobody wants to debug someone else's non-opensource code for free.

**Comment by Hrishikesh-K** [comment]
Sorry, I should have included this detail:

My repo is here: https://github.com/Hrishikesh-K/Portfolio/tree/v2
After you said, I also made a minimal example: [demo.zip](https://github.com/katspaugh/wavesurfer.js/files/5825526/demo.zip)

I figured out that by removing UIkit's CSS from the page, it was working fine. So, I went ahead and tried to find the UIkit-specific styles that were affecting this and it turns out that it was applying `max-width: 100%` to the `<canvas>` element, Resetting that part with `max-width: none` works.

Thanks for the reply and apologies once again. I should have tried this before posting it here.
